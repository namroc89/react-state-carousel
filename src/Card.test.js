import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

it("renders without crashing", function () {
  render(<Card />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<Card caption="Funny Picture" totalNum="3" />);
  expect(asFragment()).toMatchSnapshot();
});

it("matches snapshot", function () {
  const { asFragment } = render(<Card caption="Sad" totalNum="5" />);
  expect(asFragment()).toMatchSnapshot();
});
