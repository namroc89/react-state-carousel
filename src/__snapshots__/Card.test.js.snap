// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Funny Picture
    </h4>
    <img
      alt="Funny Picture"
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of 3.
    </small>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot 2`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Sad
    </h4>
    <img
      alt="Sad"
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of 5.
    </small>
  </div>
</DocumentFragment>
`;
